client/src/partials/vouchers/simple.js

```diff
@@ -39,9 +39,17 @@ function SimpleJournalVoucherController(AppCache, Vouchers, $translate, Accounts
   // load the list of accounts
   Accounts.list()
   .then(function (accounts) {
-    vm.accounts = accounts;
+    vm.accounts = removeChildren(accounts);
   });
 
+  function removeChildren(accounts) {
+    var accountsOnly = accounts.map(function (item) {
```

Comment:
A cleaner way to write this would be:
```js
function removeChildren(accounts) {
  return accounts.map(function (account) {
    delete account.children;
    return account;
  });
}
```
I personally prefer the above method because it avoids local variables and is easy to reason about.  What do you think?